WTWSMS/localisation/WtWSMS_buildings.csv

```diff
@@ -0,0 +1,55 @@
+ca_aurelian_walls;Aurelian Walls;Mur d'Aurélien;;;;;;;;;;;;x
+ca_aurelian_walls_desc;The Aurelian Walls (Latin: Muri Aureliani) are a line of city walls built between 271 AD and 275 AD in Rome, during the reign of the Roman Emperors Aurelian and Probus. They superseded the earlier Servian Wall built during the 4th century BC. The walls enclosed all the seven hills of Rome plus the Campus Martius and, on the right bank of the Tiber, the Trastevere district. The river banks within the city limits appear to have been left unfortified, although they were fortified along the Campus Martius.;Le mur d'Aurélien est une enceinte fortifiée antique protégeant la ville de Rome, en Italie, construite entre 271 et 282, sous le regne des empereurs Aurélien et Probus. Cette protection n'empecha pas plusieurs sacs de Rome au cours du Ve siecle et fut mise a contribution lors des affrontements entre les Ostrogoths et le général Bélisaire au cours de la reconquete de l'Italie par Constantinople au VIe siecle. En revanche, elle protégea efficacement Rome contre les raids sarrasins du haut Moyen Âge.;;;;;;;;;;;;x
+######Alexandria...;;;;;;;;;;;;x;;
+tp_alexandria_lighthouse_0;Lighthouse of Alexandria;Phare d'Alexandrie;;;;;;;;;;;;x
+tp_alexandria_lighthouse_0_desc;The Lighthouse of Alexandria, sometimes called the Pharos of Alexandria, was a lighthouse built by the Ptolemaic Kingdom between 280 and 247 BC which was between 120 and 137 m tall. One of the Seven Wonders of the Ancient World, for many centuries it was one of the tallest man-made structures in the world.;Le phare d'Alexandrie. Il était considéré dans l'Antiquité comme la septieme des sept merveilles du monde. Il a servi de guide aux marins pendant pres de dix-sept siecles (du IIIe siecle av. J.-C. au XIVe siecle). Sa construction aurait débuté entre -299 et -289 (la date exacte est inconnue) et duré une quinzaine d'années. Les travaux sont commencés par Ptolémée Ier mais il meurt avant la fin du chantier qui est achevé sous le regne de son fils Ptolémée II.;;;;;;;;;;;;x
+tp_alexandria_lighthouse_1;Renovated Lighthouse of Alexandria;Phare rénové d'Alexandrie;;;;;;;;;;;;x
+tp_alexandria_lighthouse_1_desc;The Lighthouse of Alexandria has seen better days, but it can be restored to its former greatness.;Le phare d'Alexandrie. Il était considéré dans l'Antiquité comme la septieme des sept merveilles du monde. Il a servi de guide aux marins pendant pres de dix-sept siecles (du IIIe siecle av. J.-C. au XIVe siecle). Sa construction aurait débuté entre -299 et -289 (la date exacte est inconnue) et duré une quinzaine d'années. Les travaux sont commencés par Ptolémée Ier mais il meurt avant la fin du chantier qui est achevé sous le regne de son fils Ptolémée II.;;;;;;;;;;;;x
```

Comment:
The issue with using Wikipedia texts is that we can't just copy them in the mod without giving credit. I prefer unique texts and link to the wiki if needed. I can write up something, with inspiration from it, but not copy-paste.